### PR TITLE
Add available balances to transfer modal

### PR DESCRIPTION
**Pull request by jacogr**
Sprawled completely out of control -

- fixed max calculations to not kill RPC (both modal & transfer app)
- fixed nonce display in extrinsics (cased by the point below)
- reworked ui-reactive to original form (just simple wrappable classes)
- add `Label` & `LabelHelp` components (see PR closed below)
- Hide/show all clickables on card non-hover to make things cleaner
- Fix onSearch bug in `AddressInput` (was not working)
- Adjusted `AddressRow` layout... swapped to it everywhere for cards
- Added `AddressInfo` for the grid-y balance display (staking & accounts)
- ... show available balances on transfer
- Pull in and closes https://github.com/polkadot-js/apps/pull/1198 (Labels completely reworked)
- Part of  https://github.com/polkadot-js/apps/issues/1208

Only "part of" since probably need to be a bit less cowboy and only remove the transfer app when we can deposit into an address anywhere. The first thing that will happen is that in the "hide universe" (without it everywhere), is, well, questions since people will get confused.

... so, boiling the frog slowly

### CODE REVIEW

**Comment by jacogr** [comment]
All the millions if extra info, only displays on hover -

image

.. and finally, transfer, is well, balance-y

image

**Comment by jacogr** [comment]
Current...

Polkadot:Substrate Portal 2019-05-25 18-21-23
Polkadot:Substrate Portal 2019-05-25 18-21-02

**Comment by jacogr** [comment]
And staking now has balances like Accounts  -

image

Ok, enough is enough. Done. (For now)

File: packages/app-accounts/src/modals/Transfer.tsx [hunk 1]
```diff
@@ -136,46 +159,43 @@ class Transfer extends React.PureComponent<Props> {
...
-            <Checks
-              accountId={address}
-              extrinsic={extrinsic}
-              isSendable
-              onChange={this.onChangeFees}
-            />
-          </Wrapper>
-        </Modal.Content>
-      </>
+      <Modal.Content className='app--account-Backup-content'>
```
Comment (Tbaut) [inline packages/app-accounts/src/modals/Transfer.tsx]: "Transfer" instead of "Backup"?

**Comment by Tbaut** [comment]
Wow you have played a lot! I find things a little overwhelming, particularly with mouse over.
I think we should be scared to space things a little and if it takes too much place, we can still have an overview and an expanded view at the click of a button.
image

Little nit, I found the address (and copy button) to overlap with the "delete" button.

File: packages/ui-app/src/LabelHelp.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+};
+
+const Wrapper = styled.div`
+  cursor: help;
+  display: inline-block;
+  line-height: 1rem;
+  margin: 0 0 0 0.25rem;
+`;
+
+export default class Labelled extends React.PureComponent<Props, State> {
```
Comment (Tbaut) [inline packages/ui-app/src/LabelHelp.tsx]: Should be `LabbelHelp` I guess. I wonder how this can work without warning as it.

**Comment by jacogr** [comment]
Yes, agreed. So couple of things -

- One major issue is the (?)'s - we should move those to the help and have the explanation there. After you know what it means, it is just noise. Maybe then with keeping the help on hover, but over the label?

- Then the copy and edit buttons... there is a reason why the damn copy was removed in the first place. (I keep on calling it "damn copy"...) Need some ideas around those where it is obvious, but not too much.

File: packages/ui-app/src/LabelHelp.tsx (hunk 2 above)
Comment (jacogr) [reply packages/ui-app/src/LabelHelp.tsx]: Ahh, doesn't matter what the class is named for use. So we do

`export { default as LabelHelp } from './LabelHelp'` - so it is actually called "default" is export.

But yes, so it works whatever it is called, but should be changed. Copy error there.

**Comment by Tbaut** [comment]
> After you know what it means, it is just noise.

Agreed, with a greeting welcome message that talks about the (?) in the top right hand corner, this help overlay could become more used. We should reduce its content to the bare minimum though.

> Maybe then with keeping the help on hover, but over the label?

Either it shows a tooltip instantly (1), and it can surely become a little annoying as the balances are all over the place and those tooltip will popup as the mouse flies above. Or we add a delay before showing (2) them and users may totally miss the functionality altogether. 

Not sure if the (1) would really be this annoying in practice, we should try.

> Need some ideas around those where it is obvious, but not too much.

Let me insist on the fact that this type of behaviour isn't new to most users, and they naturally go with the mouse on the address if they want to see it fully (we can still do it on the icon as well it can't hurt). We should try something along the line of what I proposed in https://github.com/polkadot-js/apps/issues/1176 or something like 
image

**Comment by jacogr** [comment]
So want this in as-is, then the follow-ups as well -

- tiny copy (consistent with edit as actions)
- move of balance info to help on accounts (next PR)
- all AddressSummary - AddressRow (eg. creates, next PR)

**Pull request merged**